packages/react-components/babel-preset-storybook-full-source/package.json

```diff
@@ -26,13 +26,9 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@babel/core": "^7.10.4",
+    "prettier": "2.2.1",
```

Comment:
shouldn't we caret ?

Reply:
yeah, I copied it from root package.json 🙈